extensions/storage-metadata/pkg/config/defaults/defaultconfig.go

```diff
@@ -35,11 +35,11 @@ func DefaultConfig() *config.Config {
 		Service: config.Service{
 			Name: "storage-metadata",
 		},
-		GatewayEndpoint: "127.0.0.1:9142",
-		JWTSecret:       "Pive-Fumkiu4",
-		TempFolder:      filepath.Join(defaults.BaseDataPath(), "tmp", "metadata"),
-		DataServerURL:   "http://localhost:9216/data",
-		Driver:          "ocis",
+		JWTSecret:         "Pive-Fumkiu4",
+		TempFolder:        filepath.Join(defaults.BaseDataPath(), "tmp", "metadata"),
+		DataServerURL:     "http://localhost:9216/data",
+		MachineAuthAPIKey: "change-me-please",
```

Comment:
please don't introduce any "default" / hardcoded secret after #3551

Reply:
updated while mergin master into this pr